### PR TITLE
Lib/Implement Split button#954

### PR DESCRIPTION
**Pull request by ritarora**
Closes sap/fundamental#https://github.com/SAP/fundamental/issues/954

{{short description}}

#### Test

http://localhost:4000/components/button.html

#### Changelog

**New**
Updated Documentation

screen shot 2019-01-10 at 11 37 47

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit cee79faa75787427351f2b994f6badf007697b49

https://deploy-preview-1156--fundamental.netlify.com

**Comment by ritarora** [comment]
@saadmhybris @xak Implemented based on the designs provided. Let me know if there are any sketch files for the exact specs

File: scss/components/button.scss [hunk 1]
```diff
@@ -625,3 +635,7 @@ $block: #{$fd-namespace}-button;
     }
 
 }
+.fd-div--dropdown {
```
Comment (saadmhybris) [inline scss/components/button.scss]: we can avoid this class completely by using `fd-popover` here. 

also `div` is not a good choice for class name as we don't reference HTML elements.

**Comment by xak** [comment]
Please hold this. I’d like to review the implementation.

**Comment by ritarora** [comment]
@saad-mo @xak Please review..
I have hidden the dropdown by default...will need to add JS to display it on hover

**Comment by xak** [comment]
See the last part of @saad-mo feedback. This can follow the approach of `button-group` for a much cleaner implementation. A test page must be created in addition to the documentation page. See https://github.com/SAP/fundamental/wiki/Test-Component-Guidelines for useful information.

**Comment by eboyer** [comment]
@ritarora ~ For now i'm going to close this PR for the requests for changes that @xak and @saad-mo have mentioned above. Let any of us know if you have any questions!

**Pull request closed without merging**